tests/Aspire.CommunityToolkit.Testing/Aspire.CommunityToolkit.Testing.csproj

```diff
@@ -5,8 +5,13 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <IsTestProject>false</IsTestProject>
+    <IsAspireHost>true</IsAspireHost>
```

Comment:
Do we need to make the test project an Aspire host?

Reply:
Yes, functional tests need the orchestrator to work, so both `<IsAspireHost>true</IsAspireHost>` and `<PackageReference Include="Aspire.Hosting.AppHost" IsAspireProjectResource="false" />` are necessary here.
without those following runtime errors occurred.
```
 Microsoft.Extensions.Options.OptionsValidationException : Property CliPath: The path to the DCP executable used for Aspire orchestration is required.; Property DashboardPath: The path to the Aspire Dashboard binaries is missing.
```

Reply:
But does it need to be on this project? Oh, is that because it now has test classes in it?

Reply:
Yep